smdebug/tensorflow/keras.py

```diff
@@ -353,10 +369,33 @@ def _save_tensors_post_step(self, batch, logs):
         # weights, metrics
         self._save_metrics(batch, logs)
 
+        if version.parse(tf.__version__) >= version.parse("2.0.0"):
+            model = self.model
+
+            # tf.keras lumps weights and biases together under `layer.weights`.
+            weights_coll = self.collection_manager.get(CollectionKeys.WEIGHTS)
+            bias_coll = self.collection_manager.get(CollectionKeys.BIASES)
+            for layer in model.layers:
+                for vars in layer.variables:
```

Comment:
There's some code somewhere in TF which does something similar. See if you can reuse that

Reply:
code from prepare_layers() can be reused, but the name of the tensor saved would be something like dense/weights/dense/bias:0 for bias